benchmarks/auth0.mjs

```diff
@@ -1,32 +1,14 @@
 'use strict'
 
-import { isMainThread } from 'worker_threads'
-
 import { tokens, privateKeys, publicKeys, compareSigning, compareVerifying, saveLogs } from './utils.mjs'
 
 async function runSuites() {
-  if (!isMainThread) {
-    const algorightm = process.env.CURRENT_ALGORITHM
-
-    if (process.env.CURRENT_PHASE === 'sign') {
-      compareSigning({ a: 1, b: 2, c: 3 }, algorightm, privateKeys[algorightm], publicKeys[algorightm])
-    } else {
-      compareVerifying(tokens[algorightm], algorightm, publicKeys[algorightm])
-    }
-
-    return
-  } else {
-    for (const algorightm of ['HS256', 'RS256']) {
-      process.env.CURRENT_ALGORITHM = algorightm
-
-      process.env.CURRENT_PHASE = 'sign'
-      await compareSigning({ a: 1, b: 2, c: 3 }, algorightm, privateKeys[algorightm], publicKeys[algorightm])
-      process.env.CURRENT_PHASE = 'verify'
-      await compareVerifying(tokens[algorightm], algorightm, publicKeys[algorightm])
-    }
+  for (const algorithm of ['HS256', 'RS256']) {
+    await compareSigning({ a: 1, b: 2, c: 3 }, algorithm, privateKeys[algorithm], publicKeys[algorithm])
+    await compareVerifying(tokens[algorithm], algorithm, publicKeys[algorithm])
   }
 
   await saveLogs('auth0')
 }
 
-runSuites().catch(console.error)
+runSuites()
```

Comment:
nit: let's take this one step further. top level await has been here for a long time, let's use it!